pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -189,6 +189,88 @@ func init() {
 			eventsRule(),
 		},
 	})
+
+	// sdn-controller
+	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraSDNControllerServiceAccountName},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule("get", "create", "update").Groups(networkGroup, legacyNetworkGroup).Resources("clusternetworks").RuleOrDie(),
+			rbac.NewRule("get", "list").Groups(networkGroup, legacyNetworkGroup).Resources("egressnetworkpolicies").RuleOrDie(),
+			rbac.NewRule("get", "list", "create", "delete").Groups(networkGroup, legacyNetworkGroup).Resources("hostsubnets").RuleOrDie(),
```

Comment:
Fixed in https://github.com/openshift/origin/pull/14968, added watch access to the other needed resources.